### PR TITLE
Last few changes

### PR DESCRIPTION
**Pull request by kashizui**
See commit messages.

@percyliang please review!

### CODE REVIEW

File: codalab/bundles/named_bundle.py
Comment (kashizui) [inline codalab/bundles/named_bundle.py]: The `-d` short key was conflicting with `-d` for depth in mimic.

File: codalab/lib/bundle_cli.py
Comment (kashizui) [inline codalab/lib/bundle_cli.py:673]: `metadata_arguments([MakeBundle, RunBundle])` already includes name

File: codalab/bundles/named_bundle.py
Comment (percyliang) [reply codalab/bundles/named_bundle.py]: Hmm...isn't this going to break backward compatibility?  I'd rather just disable `-d` for depth because I don't think people are likely to use that, and if they are, then `--depth` is fine.

File: codalab/lib/bundle_cli.py
Comment (percyliang) [inline codalab/lib/bundle_cli.py:805]: Still would prefer to have `cl -v` and `cl --version` work instead of having it here, since that's what people are used to typing.

**Review by percyliang — CHANGES_REQUESTED** [review]
Cool, remember to update the Wiki too (wish there were a way to update it as part of this PR)...

**Comment by kashizui** [comment]
@percyliang Made requested changes, will merge and update wiki once approved

**Review by percyliang — APPROVED** [review]
Huh, I can't check out the `hail-mary` branch anymore, but I'll trust that everything works as intended.

**Comment by kashizui** [comment]
I might have force-pushed at some point, so you could try
```
git checkout kashizui/hail-mary
git fetch
git reset --hard origin/kashizui/hail-mary
```

**Pull request merged**
